pydicer/input/test.py

```diff
@@ -1,5 +1,5 @@
 import zipfile
-import urllib
+import urllib.request
```

Comment:
I'm just curious why you had to change the urllib import here? I remember having to do this for in different project too. Does it have to do with the urllib version being used?

Reply:
Good question, this stumped me too because in theory it shouldn't make a difference. I only found this link but no one gave the reason https://stackoverflow.com/questions/35822532/i-dont-see-urllib-request-module-in-python3-throws-error

I am suspicious that it could be python versions but no clear answer unfortunately...

Reply:
Yep, it's a bit weird... but I think your change covers us this should work for all scenarios.